src/client/package.json

```diff
@@ -52,6 +52,7 @@
     "@fortawesome/react-fontawesome": "^0.1.2",
     "@glue42/desktop": "^4.5.7",
     "@glue42/office": "^1.4.9",
+    "@types/react-measure": "^2.0.5",
```

Comment:
adding react-measure to be able to change the size of the launcher window according to search controls appearing on the screen